internal/utils/templates/initial_schemas/14.sql

```diff
@@ -348,6 +394,7 @@ BEGIN
 
     IF func_is_graphql_resolve
     THEN
+        DROP FUNCTION IF EXISTS graphql_public.graphql;
```

Comment:
Where's this line coming from? This function hasn't changed since.

Reply:
It's coming from here https://github.com/supabase/infrastructure/pull/9042. I found this issue by running all infra migration scripts in self hosted docker.